pkg/api/pss_test.go

```diff
@@ -115,9 +113,10 @@ func TestPssWebsocketSingleHandlerDeregister(t *testing.T) {
 
 func TestPssWebsocketMultiHandler(t *testing.T) {
 	var (
-		pss, cl, listener = newPssTest(t, opts{})
-		u                 = url.URL{Scheme: "ws", Host: listener, Path: "/pss/subscribe/testtopic"}
-		cl2, _, err       = websocket.DefaultDialer.Dial(u.String(), nil)
+		p, publicKey, cl, listener = newPssTest(t, opts{})
+
```

Comment:
newline addition not needed

Reply:
Yes, but this way fewer lines are changed.

Reply:
Not sure if i understand what you mean. There's three lines of code on the left and there's three lines of code on the right

Reply:
This is not only place in this file with new line added.